pages/form/Suggestions.jsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from "react";
+import { Box, Heading, Text, Textarea } from "@chakra-ui/react";
+import Layout from "../../components/Layout/Layout";
+import useForm from "../../components/FormProvider";
+import {
+  BackButton,
+  ContinueButton
+} from "../../components/LinkButton/LinkButton";
+import Q7Progress from "../../public/images/progress-bar/q7-progress-dots.svg";
```

Comment:
q7-progress-dots.svg is not in the progress bar file yet. Is this something you have done locally.

Reply:
Hi @Amutha37 , ops, this PR is for progress-bar icons only, apologies. I replaced the icons and kept original name.
Goodpickup!

Reply:
Link to the Suggestions.jsx PR - (https://github.com/CodeforAustralia/council-emissions-calculator/pull/93)

Reply:
This is all good.